css/style.css

```
@@ -0,0 +1,1216 @@
+    padding: 32px 104px 64px 104px;
+    flex-direction: column;
+    align-items: center;
+    flex-shrink: 0;
+    align-self: stretch;
+    background: var(--The-julge-black, #111322);
+}
+
+.footer .frame3{
+    display: flex;
+justify-content: space-between;
+align-items: flex-start;
+align-self: stretch;
+}
+
+.footer .codeit{
+    color: #676767;
+text-align: center;
+font-family: Acme;
+font-size: 16px;
+font-style: normal;
+font-weight: 400;
+line-height: normal;
+}
+
+.footer .frame4{
+    display: flex;
+width: 181.111px;
+align-items: flex-start;
+gap: 30px;
+}
+
+.footer .frame5{
+    display: flex;
+    width: 181.111px;
+    align-items: flex-start;
+    gap: 12px;
+
+}
+
+.footer .policy{
+    color: #CFCFCF;
+font-family: Acme;
+font-size: 16px;
+font-style: normal;
+font-weight: 400;
+line-height: normal;
+}
+
+}
+
+
+
+
+
+
+/* 반응형 웹 퍼블리싱 (모바일)*/
+
+
+@media(min-width: 375px) and (max-width: 767px){
+  /* 네비 게이션 바 */
+
+  nav {
+    display: flex;
+    justify-content: center;
+    position: sticky;
+    top: 0;
+    width: 100%;
+    background-color: #edf7ff;
+    z-index: 1000;
+  }
+  
+  .toolbar{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: 9.4rem;
+    max-width: 192rem;
+    padding: 0 32px;
+  }
+  
+  
+  .toolbar a{
+    text-decoration: none;
+  }
+  
+  .toolbar a.login{
+      color: #ffffff;
+      display: flex;
+      width: 128px;
+      padding: 16px 20px;
+      justify-content: center;
+      align-items: center;
+      cursor: pointer;
+      gap: 10px;
+      border-radius: 8px;
+      background: var(--gra-purpleblue-to-skyblue, linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%));
+  }
+  
+  /* 1번 컨텐츠 */
+  
+  .bg{
+      background-color: #F0F6FF;
+      display: flex;
+  width: fixed;
+  height: fixed;
+  padding: 28px 32px 0px 33px;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 24px;
+  }
+  
+  .bg .title{
+      text-align: center;
+  font-family: Pretendard;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 80px; /* 125% */
+  word-break: keep-all;
+  width: 300px;
+  height: 250px;
+  }
+  
+  .bg .grad{
+      background: linear-gradient(91deg, #6D6AFE 17.28%, #FF9F9F 74.98%);
+      background-clip: text;
+      -webkit-background-clip: text;
+      -webkit-text-fill-color: transparent;
+  }
+  
+  .black{
+      color: #000;
+  font-family: Pretendard;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 80px;
+  
+  }
+  
+  .bg .addlink{
+      color: #ffffff;
+      display: flex;
+      width: 200px;
+      padding: 10px 16px;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  border-radius: 8px;
+  background: var(--gra-purpleblue-to-skyblue, linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%));
+  }
+  
+  .bg .img1{
+    width: 100%;
+    height: 100%;
+  overflow: hidden;
+  }
```

> Mobile 사이즈에서 내부 요소들의 너비는 기기의 너비가 커지는 만큼 커지지만 400px 넘지않도록 max-width요소를 주었는데도 잘 되지않아 원래 코드를 제출하였습니다.

해당 코드를 제출해주시지 않아 정확히 어떤 문제인지는 모르겠지만 이미지의 최대 크기는 이곳에 적용하면 말씀하신대로 400px까지만 커지도록 구현할 수 있어요.

```suggestion
  .bg .img1{
    width: 100%;
    height: 100%;
    max-width: 400px
    overflow: hidden;
  }
```

---

넵 한번 해보겠습니다. 감사합니다